taxpose/datasets/enums.py

```diff
@@ -0,0 +1,14 @@
+from enum import Enum
```

Comment:
this is fine - but I don't really like the ObjectClass / phase stuff anymore. doesn't really match my new mental model of things.

Reply:
fine for now tho

Reply:
I see, mostly did this shifting around to prevent a circular import